lib/block-supports/typography.php

```diff
@@ -481,12 +481,13 @@ function gutenberg_get_typography_font_size_value( $preset, $should_use_fluid_ty
 	$fluid_settings = isset( $typography_settings['fluid'] ) && is_array( $typography_settings['fluid'] ) ? $typography_settings['fluid'] : array();
 
 	// Defaults.
-	$default_maximum_viewport_width   = '1600px';
-	$default_minimum_viewport_width   = '768px';
-	$default_minimum_font_size_factor = 0.75;
-	$default_scale_factor             = 1;
-	$has_min_font_size                = isset( $fluid_settings['minFontSize'] ) && ! empty( gutenberg_get_typography_value_and_unit( $fluid_settings['minFontSize'] ) );
-	$default_minimum_font_size_limit  = $has_min_font_size ? $fluid_settings['minFontSize'] : '14px';
+	$default_maximum_viewport_width       = '1600px';
```

Comment:
This may be more useful referencing the layout.wideSize value from theme.json, so that fonts stops growing when wide width elements are maxed out—as the `wideSize` size  is used as the page width in many cases.

Reply:
Notice how the layout keeps shifting (as the fontSize is still growing), even when the max size is reached: 

https://user-images.githubusercontent.com/1813435/231865575-f6e2d4c9-abff-47dd-ba0c-fe437820de76.mp4

Reply:
Thanks! I agree it's definitely something we can and probably should do. It was slated for the [original incarnation of fluid font sizes](https://github.com/WordPress/gutenberg/pull/39529#issuecomment-1113132610), but I think it fell off radar since we wanted to test the fallback value. From memory it tested pretty well.

I think a manageable approach would be to throw up a new PR using this PR as the base.

👍